Makefile

```diff
@@ -16,5 +19,12 @@ check:
 	PYTHONPATH=$(CURDIR) PYTHONDONTWRITEBYTECODE=1 python3 -m pytest --verbose --showlocals --cov=packit --cov-report=term-missing $(TEST_TARGET)
 
 check_in_container: test_container
-	rsync -a $(CURDIR)/ /tmp/packit
+	rsync -a $(CURDIR)/ $(PACKIT_PATH)
```

Comment:
can we make it work without this hack? it drives me nuts all the time

Reply:
yep, removed, but probably we had to disable CentOS CI, because it will fail.
We could done as separate issue https://github.com/packit-service/packit/issues/473